types/fontkit/fontkit-tests.ts

```diff
@@ -24,37 +30,50 @@ const f = fontkit.openSync("fonts/a-font.ttf");
 // ----------------
 // API: Font.layout
 // ----------------
-const { glyphs, positions } = f.layout("Hello World!");
-
-const res = [];
-for (let i = 0; i < glyphs.length; i++) {
-    const glyph = glyphs[i];
-    const pos = positions[i];
-    let x = `${glyph.id}`;
-    if (pos.xOffset || pos.yOffset) {
-        x += `@${pos.xOffset},${pos.yOffset}`;
-    }
+if (!isCollection(f)) {
+    const { glyphs, positions } = f.layout("Hello World!");
+    const res = [];
+    for (let i = 0; i < glyphs.length; i++) {
+        const glyph = glyphs[i];
+        const pos = positions[i];
+        let x = `${glyph.id}`;
+        if (pos.xOffset || pos.yOffset) {
+            x += `@${pos.xOffset},${pos.yOffset}`;
+        }
 
-    x += `+${pos.xAdvance}`;
-    res.push(x);
-}
+        x += `+${pos.xAdvance}`;
+        res.push(x);
+    }
 
-res.join("|");
+    res.join("|");
 
-const dims = [];
-for (const glyph of glyphs) {
-    const bbox = glyph.bbox;
-    let dim = `${glyph.id}`;
-    dim += ` w:${bbox.width} h:${bbox.height}`;
-    dims.push(dim);
+    const dims = [];
+    for (const glyph of glyphs) {
+        const bbox = glyph.bbox;
+        let dim = `${glyph.id}`;
+        dim += ` w:${bbox.width} h:${bbox.height}`;
+        dims.push(dim);
+    }
+    dims.join("|");
 }
-dims.join("|");
 
 // -----------------
 // API: Font['hhea']
 // -----------------
 openV2.then(font => {
-    font["hhea"].version; // $ExpectType number
-    font.getGlyph; // $ExpectType (glyphId: number, codePoints?: number[] | undefined) => Glyph
-    font.getGlyph(0); // $ExpectType Glyph
+    if (!isCollection(font)) {
+        font["hhea"].version; // $ExpectType number
+        font.getGlyph; // $ExpectType (glyphId: number, codePoints?: number[] | undefined) => Glyph
+        font.getGlyph(0); // $ExpectType Glyph
+    }
 });
+
+// ------------------
+// API: FontCollection
+// -----------------
+openV2.then(font => {
+    if (isCollection(font)) {
+        font.fonts; // $ExpectType Font[]
+        font.getFont("hhe") // $ExpectType Font | null
+    }
+})
```

Comment:
open returns either a Font or a FontCollection. It can be distinguished by `type`.